src/jsonpatcherproxy.js

```diff
@@ -76,15 +107,29 @@ var JSONPatcherProxy = (function() {
         return Reflect.get(target, propKey, receiver);
       },
       set: function(target, key, receiver) {
-        const distPath =
-          path + '/' + JSONPatcherProxy.escapePathComponent(key.toString());
+
```

Comment:
And I have been calling this name stupid since I built the thing. Turns out it's my fault -_-